misc/security-and-privacy.md

```diff
@@ -0,0 +1,96 @@
+# [Self-Review Questionnaire: Security and Privacy](https://w3ctag.github.io/security-questionnaire/)
+
+This questionnaire has [moved](https://w3ctag.github.io/security-questionnaire/).
+
+# Security and Privacy Self-review for IMSC-HRM
+
+This is an edited version of the security and privacy questionnaire template at
+https://raw.githubusercontent.com/w3ctag/security-questionnaire/main/questionnaire.markdown as linked from
```

Comment:
```suggestion
https://w3ctag.github.io/security-questionnaire/ as linked from
```

Reply:
I think the better link to use is https://github.com/w3ctag/security-questionnaire/blob/main/questionnaire.markdown if anything, but I'm not sure what's wrong with the original one?